firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigContainerTest.java

```diff
@@ -350,4 +464,17 @@ private static ImmutableMap<String, String> generateP13nMetadata(
     return ImmutableMap.of(
         PERSONALIZATION_ID, p13nId, ARM_INDEX, armIndex, CHOICE_ID, choiceId, GROUP, group);
   }
+
+  private static JSONArray generateRolloutMetadata(int numberOfMetadata) throws JSONException {
```

Comment:
not blocking - we could probably improve the pattern for creating metadata (both ABT and Rollouts) since it's not super clear what it means to call `generateRolloutMetadata(1)` in a test.

Maybe an improvement is calling this `generateNRolloutMetadataEntries(int numberOfMetadata)`?